libs/react-components/community/src/components/form/file-upload/file-upload.stories.tsx

```diff
@@ -89,16 +89,14 @@ export const PdfAndTxtOnly: Story = {
 };
 
 const MaxSizeTemplate: StoryFn<ComponentProps<typeof FileUpload>> = (args) => {
-  const [invalidSizeFiles, setInvalidSizeFiles] = useState<File[]>([]);
-
+  const [helperText, setHelperText] = useState<string | undefined>();
+  const handleInvalidFiles = (files: File[]) => {
+    setHelperText(`File(s) ${files.map((file) => `'${file.name}'`).join(', ')} are too big`);
+  };
   return (
     <VerticalSpacing>
-      <FileUpload {...args} onInvalidSize={setInvalidSizeFiles} />
-      {invalidSizeFiles.length > 0 && (
-        <Notification type="error">
-          File(s) {invalidSizeFiles.map((file) => `'${file.name}'`).join(', ')} are too big
-        </Notification>
-      )}
+      <FileUpload {...args} onInvalidSize={handleInvalidFiles} onChange={() => setHelperText(undefined)} />
+      {helperText && <FormHelper text={helperText} type="error" />}
```

Comment:
Should be used from FileUpload propertie `helper` not rendered separated.

Reply:
Done.